app-admin/u-boot-menu/autobuild/postinst

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -e
+
+if [ "$1" = "triggered" ]
+then
+	/usr/bin/u-boot-update
+fi
+
+if [ "$1" = "configure" ]
+then
+	# Create an initial extlinux.conf if directory not already present
+	if [ -z "$2" ] && [ ! -e "/boot/extlinux" ]
+	then
+		/usr/bin/u-boot-update
+	fi
+fi
+
+#DEBHELPER#
```

Comment:
What is this for?

Reply:
I copied this directly from the postinst in a Debian package, and ~I don't know either~.

Reply:
So remove it. Any line of code that we can't explain is one line too much.